source/options.tsx

```diff
@@ -59,13 +59,16 @@ function reportStatus({tokenType, error, text, scopes}: Status): void {
 	}
 }
 
-async function getTokenScopes(personalToken: string): Promise<string[]> {
+function getApiURL(): string {
```

Comment:
Let's prefer Url unless the function returns a `new URL` instance (which in this case it doesn't)